Documentation/Storage-Configuration/Advanced/ceph-configuration.md

```diff
@@ -47,12 +47,24 @@ kubectl apply -f common.yaml -f operator.yaml -f cluster.yaml # add other files
 
 ## Deploying a second cluster
 
-If you wish to create a new CephCluster in a different namespace than `rook-ceph` while using a single operator to manage both clusters execute the following:
+If you wish to create a new CephCluster in a different namespace than `rook-ceph` while using a
+single operator to manage both clusters change `SECOND_ROOK_CLUSTER_NAMESPACE`. The `common-second-cluster`
+manifests is commented to allow for easy `sed` replacements.
 
 ```console
 cd deploy/examples
+export SECOND_ROOK_CLUSTER_NAMESPACE="rook-ceph"
```

Comment:
Let's keep the example the same for the secondary namespace. The primary namespace is usually `rook-ceph`.
```suggestion
export SECOND_ROOK_CLUSTER_NAMESPACE="rook-ceph-secondary"
```